ProcessLib/ThermoMechanicalPhaseField/ThermoMechanicalPhaseFieldProcess-impl.h

```diff
@@ -34,14 +34,12 @@ ThermoMechanicalPhaseFieldProcess<DisplacementDim>::ThermoMechanicalPhaseFieldPr
     ThermoMechanicalPhaseFieldProcessData<DisplacementDim>&& process_data,
     SecondaryVariableCollection&& secondary_variables,
     NumLib::NamedFunctionCaller&& named_function_caller,
-    bool const use_monolithic_scheme,
     int const mechanics_related_process_id,
     int const phase_field_process_id,
     int const heat_conduction_process_id)
     : Process(mesh, std::move(jacobian_assembler), parameters,
               integration_order, std::move(process_variables),
-              std::move(secondary_variables), std::move(named_function_caller),
-              use_monolithic_scheme),
+              std::move(secondary_variables), std::move(named_function_caller)),
```

Comment:
By changing it to
``
std::move(secondary_variables), std::move(named_function_caller), false),
``
the tests can be run.

Reply:
fixed.